doc/reference/guidelines.rst

```diff
@@ -255,22 +304,24 @@ Inside these 3 groups, the imported lines are alphabetically sorted.
 Idioms
 -------
 
-- Prefer ``%`` over ``.format()``, prefer ``%(varname)`` instead of position (This is better for translation)
-- Try to avoid generators and decorators
-- Always favor *Readability* over *conciseness* or using the language features or idioms.
+- Prefer ``%`` over ``.format()`` (when only one variable to replace in a string), prefer ``%(varname)`` instead of position (when multiple variables are to be replaced in a string). This makes the translation easier for the translators community.
+- Avoid to create generators and decorators: only use the ones provide by the Odoo API.
+- Always favor *readability* over *conciseness* or using the language features or idioms.
 - Use list comprehension, dict comprehension, and basic manipulation using ``map``, ``filter``, ``sum``, ... They make the code easier to read.
 - The same applies for recordset methods : use ``filtered``, ``mapped``, ``sorted``, ...
 - Each python file should have ``# -*- coding: utf-8 -*-`` as first line
-- Use the ``UserError`` defined in ``openerp.exceptions`` instead of overriding ``Warning``, or find a more appropriate exception in *exceptions.py*
+- Use the ``UserError`` defined in ``openerp.exceptions`` instead of overriding ``Warning``, or find a more appropriate exception in *exceptions.py* (only from v9).
 - Document your code (docstring on methods, simple comments for the tricky part of the code)
 - Use meaningful variable/class/method names
-
+- Every method used to compute data for a 'stat button' should use a `read_group` or a SQL query. This aims to improve performance (by computing data in only on query)
 
 
 Symbols
 -------
-
-- Odoo Python Class : use camelcase for code in api v8, underscore lowercase notation for old api.
+- Model name (using the dot notation, prefix by the module name) :
+    - When defining an Odoo Model : use singular form of the name (`res.partner` and `sale.order` instead of `res.partnerS` and `saleS.orderS`)
```

Comment:
And the first word should be the "main module" it relates to, such as `account.x` or `sale.x`.